manifest.json

```diff
@@ -1,10 +1,10 @@
 {
-  "serial": 2022052502,
-  "hash": "6/q/WxFHUb/lCat9Rv8/5EamkXm20acqiJ/l5Fdr9jQ=",
+  "serial": 2022060701,
+  "hash": "HspvNZrBJ7JcQkDcYq1YcSWz2GRbf6MtiQByTY1fcnI=",
   "signatures": [
     {
       "address": "13YqCei6dP2ibq2DAy81NmU8FYwwzo5HLKXFA3yM3Rvq9WxDfKF",
-      "signature": "rGUK2I8VPSqZrJqWvM6UbXIe6+m95bEDn7XZXJr382XbSux8BBdV8S43ZkYmRt1+MWjoYuM8eNudnbwLXTGiCA=="
+      "signature": ""
     },
     {
       "address": "14egjs9cSxt4jnaZTYQE4NemQVXL6qtayqNNpy3VJPSS6AAcaUq",
```

Comment:
Signature for `14egjs9cSxt4jnaZTYQE4NemQVXL6qtayqNNpy3VJPSS6AAcaUq`

`kA4fH4/vdyBDVOET5v09Mo6MXQjHnwxSP8JGWrCP9Fvuy4EYHQb5ojxn16dddhVNXa8lBv6X7fE5CfHX0AOPCA==`

Reply:
Signature for `13RKFFy5qmibyaS6N3Uw18UBDwGhhsXpk6jDn7SMenfDrQ6o6eh`
`"signature": "Fcg+/wieP7TOgF57mhNBUqWpccXEhh7agilwwpOHJVm4G98wT6yQADT49mzBly/vlqxjKNEtbLXt1sfJ7vYEBA=="`